Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -21,7 +21,9 @@
     '2023D2Timing',
     '2023D2TimingPU',
     '2023D3Timing',
-    '2023D3TimingPU'
+    '2023D3TimingPU',
+    '2023D4',
+    '2023D4PU',
```

Comment:
ok just did in my last commit, but seems I broke runthematrix when I was trying to test, must have introduced a typo soemwhere in my last commit